error opening help when in file explorer - AssertionError: Non-unique window ID

``` sh
% vim .
:h help<CR>
```

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/Cellar/python/2.7.5/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/Powerline-beta-py2.7.egg/powerline/vim.py", line 138
, in statusline
    window, window_id, winnr = self.win_idx(window_id) or (None, None, None)
  File "/usr/local/Cellar/python/2.7.5/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/Powerline-beta-py2.7.egg/powerline/vim.py", line 133
, in win_idx
    assert r is None, "Non-unique window ID"
AssertionError: Non-unique window ID
E858: Eval did not return a valid python object
```

--- a/powerline/vim.py
+++ b/powerline/vim.py
@@ -101,6 +101,8 @@ class VimPowerline(Powerline):
 			for window in vim.windows:
 				try:
 					curwindow_id = window.vars['powerline_window_id']
+					if r is not None and curwindow_id == window_id:
+						raise KeyError
 				except KeyError:
 					curwindow_id = self.last_window_id
 					self.last_window_id += 1
@@ -109,7 +111,6 @@ class VimPowerline(Powerline):
 				if window.options['statusline'] != statusline:
 					window.options['statusline'] = statusline
 				if curwindow_id == window_id if window_id else window is vim.current.window:
-					assert r is None, "Non-unique window ID"
 					r = (window, curwindow_id, window.number)
 			return r
 	else:
@@ -120,7 +121,7 @@ class VimPowerline(Powerline):
 			r = None
 			for winnr, window in zip(count(1), vim.windows):
 				curwindow_id = self._vim_getwinvar(winnr, 'powerline_window_id')
-				if curwindow_id:
+				if curwindow_id and not (r is not None and curwindow_id == window_id):
 					curwindow_id = int(curwindow_id)
 				else:
 					curwindow_id = self.last_window_id
@@ -130,7 +131,6 @@ class VimPowerline(Powerline):
 				if self._vim_getwinvar(winnr, '&statusline') != statusline:
 					self._vim_setwinvar(winnr, '&statusline', statusline)
 				if curwindow_id == window_id if window_id else window is vim.current.window:
-					assert r is None, "Non-unique window ID"
 					r = (window, curwindow_id, winnr)
 			return r
 

